src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -123,7 +117,7 @@ const std::shared_ptr<AbstractLQPNode>& SQLPipelineStatement::get_optimized_logi
       if (lqp_is_validated(plan) == (_use_mvcc == UseMvcc::Yes)) {
         // Copy the LQP for reuse as the LQPTranslator might modify mutable fields (e.g., cached output_expressions)
         // and concurrent translations might conflict.
-        _optimized_logical_plan = plan->deep_copy();
+        _optimized_logical_plan = {true, plan->deep_copy()};
```

Comment:
Like in the other PR proposed, I like this style, as I am not sure what is actually being set to true here. Maybe you deem it a good idea as well :)
```suggestion
        _optimized_logical_plan = {/* bool variable changed here =*/ true, plan->deep_copy()};
```

Reply:
I added a comment on why it is set to true